.github/workflows/nudge-stale.yml

```diff
@@ -0,0 +1,31 @@
+name: Nudge stale PRs
+'on':
+  workflow_dispatch:
+  schedule:
+    # Every 12 hours
+    - cron: '0 */12 * * *'
+jobs:
+  auto-merge:
```

Comment:
Should this job be named differently here?

Reply:
Oops